components/html-embed/index.js

```diff
@@ -0,0 +1,37 @@
+export default class HtmlEmbed extends wp.element.Component {
```

Comment:
I'm not sure I understand what this component is doing, could you clarify a bit? Thanks